<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PortTab</class>
 <widget class="QWidget" name="PortTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="portComboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">padding-left: 5pt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="portSearchPushButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="portConnectPushButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="baudRateComboBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">padding-left: 5pt;</string>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="currentIndex">
    <number>6</number>
   </property>
   <item>
    <property name="text">
     <string>9600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>28800</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>38400</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>57600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>76800</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>115200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>230400</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>460800</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>921600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1000000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2000000</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="baudRateLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Buad Rate</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
